server/src/resolvers/roomsResolver.ts

```diff
@@ -0,0 +1,125 @@
+import {EquipmentRepository} from "../repositories/Equipment/EquipmentRepository";
+import {UserRepository} from "../repositories/Users/UserRepository";
+import {RoomRepo} from "../repositories/Rooms/RoomRepository";
+
+const er = new EquipmentRepository();
+const ur = new UserRepository();
+const rr = new RoomRepo();
+
+const RoomResolvers = {
+    Query: {
+        rooms: async (_: any, args: any, context: any) => {
+            try {
```

Comment:
these try/catch statements stop errors from making it to the graphql playground. I've seen it error and return null for an object. Not sure on whether or not having them here is a good idea. Let me know what you think